packages/unlock-js/src/PublicLock/utils/parseLockGetters.js

```diff
@@ -66,9 +67,20 @@ export default async function (address, provider) {
     )
     const erc20Symbol = await getErc20TokenSymbol(update.tokenAddress, provider)
 
+    // get lock allowance of itself (for v10 recurring)
+    const erc20LockAllowance = await getAllowance(
```

Comment:
Since this is only for v10 can we move that into the v10's `getLock` so that we don't slow things down for other locks?